HELP.md

```
@@ -0,0 +1,31 @@
+# Read Me First
+The following was discovered as part of building this project:
+
+* The original package name 'com.first-ticket.config-server' is invalid and this project uses 'com.first_ticket.config_server' instead.
+
+# Getting Started
+
+### Reference Documentation
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)